app/ts/utils/ethereum.ts

```diff
@@ -49,6 +49,23 @@ export interface IUnsignedTransaction1559 {
 	}[]
 }
 
+export interface OptionalIUnsignedTransaction1559 {
+	readonly type: '1559'
+	readonly from: bigint
+	readonly chainId: bigint
+	readonly nonce: bigint
+	readonly maxFeePerGas?: bigint
+	readonly maxPriorityFeePerGas: bigint
+	readonly gasLimit: bigint
+	readonly to: bigint | null
+	readonly value: bigint
+	readonly input: Uint8Array
+	readonly accessList: readonly {
+		readonly address: bigint
+		readonly storageKeys: readonly bigint[]
+	}[]
+}
```

Comment:
Consider changing this to a type and just doing something like:
```ts
export type OptionalIUnsignedTransaction1559 = Omit<IUnsignedTransaction1559, 'maxFeePerGas'> & Optional<IUnsignedTransaction1559>
```
Note, above code is pseudocode to give you an idea, I'm not sure what the best way to achieve this is though.  Happy to workshop it in Discord.

Reply:
its not actually used anywhere :D. I removed it